lib/_tls_wrap.js

```diff
@@ -411,6 +411,7 @@ function TLSSocket(socket, opts) {
   net.Socket.call(this, {
     handle: this._wrapHandle(wrap),
     allowHalfOpen: socket ? socket.allowHalfOpen : tlsOptions.allowHalfOpen,
```

Comment:
As written, `tlsOptions.allowHalfOpen` will never be set - right? Should I also add that to the TLSSocket call in `tlsConnectionListener`?